workers/Worker-location.any.js

```diff
@@ -0,0 +1,14 @@
+// META: global=!default, dedicatedworker, sharedworker
+test(() => {
+  assert_equals(String(WorkerLocation), "function WorkerLocation() { [native code] }");
+  assert_true(location instanceof Object);
+  assert_equals(location.href, "http://web-platform.test:8001/workers/Worker-location.any.worker.js")
```

Comment:
This looks broken...

Reply:
Ah, yes, but somehow still passing lint. @kjd564 can you try replacing the URL here with https://web-platform-tests.org/writing-tests/server-features.html? That will require renaming the file to `Worker-location.sub.any.js`, and fetch/api/basic/integrity.sub.any.js might serve as a good example to copy from.

Reply:
Done! Sent out a cl. Thanks for the pointer.